packages/react-components/src/lib/date-range-picker/stories/DateRangePicker.stories.tsx

```diff
@@ -15,3 +18,182 @@ export const DateTime: CSFStory<JSX.Element> = () => (
 DateTime.story = {
   name: 'Date time range',
 };
+
+export const DateNoFutureLimitTime: CSFStory<JSX.Element> = () => (
+  <DateTimeRangePicker
+    disableFuture={false}
+    onChange={action('date changed')}
+  />
+);
+
+DateNoFutureLimitTime.story = {
+  name: 'Date time range no future limit',
+};
+
+export const DateMinMaxLimitTime: CSFStory<JSX.Element> = () => (
+  <DateTimeRangePicker
+    onChange={action('date changed')}
+    minDate={new Date('09/01/2020')}
+    maxDate={new Date('11/01/2020')}
```

Comment:
use `current date` for minDate and `new date + 5 days` so it is relevant for every date in the future.

Reply:
Done, added +/- delta for current date for min and max dates